build.sbt

```diff
@@ -7,6 +7,7 @@ scalaVersion := scala212
 
 val commonSettings = Seq(
   organization := "com.lightbend.akka.grpc",
+  resolvers += "Staging Repo" at "https://oss.sonatype.org/content/repositories/comtypesafe-2276",
```

Comment:
Leftover?

Reply:
yeah this PR was also a 'smoke test' before we actually synced to maven central :D

Reply:
But you can maybe add `https://oss.sonatype.org/content/repositories/releases/` so that we won't need to wait for Maven Central sync.